compute/instances.go

```diff
@@ -103,6 +104,39 @@ func (gmi *GetInstanceInput) Validate() error {
 	return nil
 }
 
+//
+type CountInstancesInput struct{}
```

Comment:
is this even worth adding? There are some other input structs in other packages but those are areas in which filters can occur

Reply:
I asked myself the same question "should there be an input parameter here at all?", then I looked at some other examples e.g. https://github.com/joyent/triton-go/blob/master/account/keys.go#L41 (and few others with similar concept) and decided to follow the canon.

Should we remove it?

Reply:
on second thoughts, maybe it's best to leave it for now - it may allow us to count instances based on a filter later :)

Reply:
I am happy to remove it. Since we are using `HEAD` against `/:account/machines`, there is no way to filter on anything - as there is no body aside of headers, and I use `X-Resource-Count` header to get the machines count back. :)

Reply:
ok let's remove it then :)

Reply:
Done!